tests/e2e/ctl_v3_auth_security_test.go

```diff
@@ -27,10 +27,10 @@ import (
 
 // TestAuth_CVE_2021_28235 verifies https://nvd.nist.gov/vuln/detail/CVE-2021-28235
 func TestAuth_CVE_2021_28235(t *testing.T) {
-	testCtl(t, authTest_CVE_2021_28235, withCfg(*e2e.NewConfigNoTLS()), withLogLevel("debug"))
+	testCtl(t, authTestCVE2021_28235, withCfg(*e2e.NewConfigNoTLS()), withLogLevel("debug"))
 }
 
-func authTest_CVE_2021_28235(cx ctlCtx) {
+func authTestCVE2021_28235(cx ctlCtx) {
```

Comment:
I was hesitant about this rename. However, the original test case name retains the underscores to split the CVE name properly (`TestAuth_CVE_2021_28235`).